### PR TITLE
Account rename 1: move API /config/tenants => /accounts

### PR DESCRIPTION
**Pull request by vimukthi-git**
<This is intended as a guide when describing-reviewing pull requests for go-centrifuge>
Issue Link: #617

### CODE REVIEW

File: protobufs/accounts/service.proto [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+option go_package = "accountpb";
+option java_multiple_files = true;
+option java_outer_classname = "ServiceProto";
+option java_package = "com.account";
+
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+// AccountService allows to manage the node and tenant configuration of the node
```
Comment (vimukthi-git) [inline protobufs/accounts/service.proto]: shit, this needs to change :)

File: config/configstore/handler.go [hunk 2]
```diff
@@ -23,8 +22,13 @@ func GRPCHandler(svc config.Service) configpb.ConfigServiceServer {
 	return &grpcHandler{service: svc}
 }
 
-func (h grpcHandler) deriveAllTenantResponse(cfgs []config.TenantConfiguration) (*configpb.GetAllTenantResponse, error) {
-	response := new(configpb.GetAllTenantResponse)
+// GRPCAccountHandler returns an implementation of configpb.ConfigServiceServer
```
Comment (vimukthi-git) [inline config/configstore/handler.go]: change

File: api/service.go [hunk 3]
```diff
@@ -99,6 +100,13 @@ func registerServices(ctx context.Context, cfg Config, grpcServer *grpc.Server,
 		return err
 	}
 
+	// account api
```
Comment (mikiquantum) [inline api/service.go]: Does it mean we do not need the above `config api` registration right?

File: protobufs/accounts/service.proto [hunk 4]
```diff
@@ -0,0 +1,91 @@
+syntax = "proto3";
```
Comment (mikiquantum) [inline protobufs/accounts/service.proto]: folder name is more standard in singular `protobufs/account/service.proto`

File: api/service.go (hunk 3 above)
Comment (vimukthi-git) [reply api/service.go]: we have two apis since `GET /config` endpoint is not removed and its a separate API from now on.

Comment (mikiquantum) [reply api/service.go]: why do we need the `GET` endpoint exposed if we read from the file always?

Comment (vimukthi-git) [reply api/service.go]: I felt its nicer to be able to see the config using an API, since we had the API thought of not leaving it as it is.

**Pull request merged**